estuary/models/freshmaker.py

```diff
@@ -20,9 +20,36 @@ class FreshmakerEvent(EstuaryStructuredNode):
     state_reason = StringProperty()
     triggered_by_advisory = RelationshipTo(
         '.errata.Advisory', 'TRIGGERED_BY', cardinality=ZeroOrOne)
-    triggered_container_builds = RelationshipTo('.koji.ContainerKojiBuild', 'TRIGGERED')
+    successful_koji_builds = RelationshipTo('.koji.ContainerKojiBuild', 'TRIGGERED')
+    requested_builds = RelationshipTo('.FreshmakerBuild', 'TRIGGERED')
 
     @property
     def display_name(self):
         """Get intuitive (human readable) display name for the node."""
         return 'Freshmaker event {0}'.format(self.id_)
+
+
+class FreshmakerBuild(EstuaryStructuredNode):
+    """Definition of a Freshmaker build in Neo4j."""
+
+    id_ = UniqueIdProperty(db_property='id')
+    build_id = IntegerProperty(unique=True, required=True)
```

Comment:
My preference it to remove `required` to all of these properties because it affects the usability of `get_or_create`. For instance, if I need to add a relationship to a `FreshmakerBuild` node and all I have is the ID, normally I'd first start by either getting the build or creating it with `FreshmakerBuild.get_or_create({'id_': 12345})`, but if all those other properties are required, then `get_or_create` will require them also.

All the required properties also determine the uniqueness of a node in `get_or_create`, so if for some reason the `url` property changes, and I run `get_or_create` with all the same information except for the URL being different, an exception will be raised saying that a new `FreshmakerBuild` was being created, but it violates the unique constraint on `id_`.